data/json/items/generic/casing.json

```diff
@@ -22,6 +22,7 @@
   {
     "id": "22_casing",
     "copy-from": "casing",
+	"looks_like": "9mm_casing",
```

Comment:
Seems all the new lines are missing a tab over to align them with the other values, [linting](http://dev.narc.ro/cataclysm/format.html) should fix that ^_^

Reply:
Thanks for noticing.